### PR TITLE
Improve logout() LocalAuth logic

### PR DESCRIPTION
**Pull request by chrishubert**
# PR Details

Improve logout() LocalAuth logic

## Description

Wait for browser to close to prevent Exception on Windows

## Related Issue

[Another attempt with try/catch](https://github.com/pedroslopez/whatsapp-web.js/pull/2118)

## Motivation and Context

Specific to Windows OS, where Chromium has to be terminated to properly delete the folder

## How Has This Been Tested

- Tested on Windows
- Added a delay before the function to ensure that even if the browser is already closed, the function still processes

## Types of changes

- [ ] Dependency change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

File: src/authStrategies/LocalAuth.js [hunk 1]
```diff
@@ -44,7 +44,8 @@ class LocalAuth extends BaseAuthStrategy {
 
     async logout() {
         if (this.userDataDir) {
-            return (fs.rmSync ? fs.rmSync : fs.rmdirSync).call(this, this.userDataDir, { recursive: true });
+            await this.client.pupBrowser.close();
```
Comment (pedroslopez) [inline src/authStrategies/LocalAuth.js:47]: Any browser manipulation should be done on the `Client` - could we move this to the client's logout function?

Comment (chrishubert) [reply src/authStrategies/LocalAuth.js:47]: @pedroslopez, thanks for the feedback - let me give it a try.

Comment (PurpShell) [reply src/authStrategies/LocalAuth.js:47]: @chrishubert this is still not implemented?

Comment (PurpShell) [reply src/authStrategies/LocalAuth.js]: I've taken the liberty of changing this myself to fix this more quickly.

**Review by purpshell — APPROVED** [review]

**Comment by chrishubert** [comment]
Awesome, thanks @PurpShell ! I was planning to work on it during the weekend – you beat me to it 🚀

**Review by tuyuribr — APPROVED** [review]

**Comment by shirser121** [comment]
Why do we not merge this?

**Comment by pedrodamiani21** [comment]
here it stopped updating the status automatically when connecting or disconnecting after this update.

**Comment by Ndohjapan** [comment]
@pedrodamiani21 same here

**Pull request merged**